scico/test/linop/test_linop.py

```diff
@@ -519,49 +519,6 @@ def sumtestobj(request):
     yield SumTestObj(request.param)
 
 
-sum_axis = [
```

Comment:
Tests for old `Sum` linop removed, but not replaced with test for the new version?

Reply:
I believe they are redundant with tests for `snp.sum`, which is tested over in test_numpy.py.

Reply:
But there are tests for the new Sum and Transpose linops. Why does a similar argument not apply to them?